### DISCUSSION TITLE
Remove detected fields in Loki

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#45320] [Observability]
Detected fields are source of confusion - what are they, what actions can they do, why are they different than labels. Moreover Loki now supports parsing and therefore users can use parsers to interact with the content of logs. Removing detected filed and moving all detected-filed-features to log labels could improve experience and make it more straight forward to interact with logs.

**Comment by connorlindsey** [comment]
Some possible options:
- Only show a derived field if it's not already in the labels
- Add a config option to hide/show derived labels
